Build system disables format warnings via -Wno-format, this causes gcc to emit
an error as it conflicts with -Werror=format-security, which we set in abuild's
defaults.

diff -upr a/o2image/Makefile b/o2image/Makefile
--- a/o2image/Makefile	2021-05-26 09:00:45.000000000 +0200
+++ b/o2image/Makefile	2024-07-11 20:58:17.143741616 +0200
@@ -2,7 +2,7 @@ TOPDIR = ..
 
 include $(TOPDIR)/Preamble.make
 
-WARNINGS = -Wall -Wstrict-prototypes -Wno-format -Wmissing-prototypes \
+WARNINGS = -Wall -Wstrict-prototypes -Wmissing-prototypes \
            -Wmissing-declarations
 
 CFLAGS += $(WARNINGS)
diff -upr a/o2info/Makefile b/o2info/Makefile
--- a/o2info/Makefile	2021-05-26 09:00:45.000000000 +0200
+++ b/o2info/Makefile	2024-07-11 20:58:12.563729681 +0200
@@ -2,7 +2,7 @@ TOPDIR = ..
 
 include $(TOPDIR)/Preamble.make
 
-WARNINGS = -Wall -Wstrict-prototypes -Wno-format -Wmissing-prototypes \
+WARNINGS = -Wall -Wstrict-prototypes -Wmissing-prototypes \
            -Wmissing-declarations
 
 CFLAGS += $(WARNINGS)
diff -upr a/o2monitor/Makefile b/o2monitor/Makefile
--- a/o2monitor/Makefile	2021-05-26 09:00:45.000000000 +0200
+++ b/o2monitor/Makefile	2024-07-11 20:58:21.397086034 +0200
@@ -4,7 +4,7 @@ include $(TOPDIR)/Preamble.make
 
 SBIN_EXTRA = o2hbmonitor
 
-WARNINGS = -Wall -Wstrict-prototypes -Wno-format -Wmissing-prototypes \
+WARNINGS = -Wall -Wstrict-prototypes -Wmissing-prototypes \
            -Wmissing-declarations
 
 CFLAGS += $(WARNINGS)
